src/gccrs/args.rs

```diff
@@ -1,14 +1,14 @@
 //! This module interprets arguments given to `rustc` and transforms them into valid
 //! arguments for `gccrs`.
 
-use std::path::PathBuf;
+use std::{path::PathBuf, process::Command};
 
 use getopts::{Matches, Options};
 
 use super::{Error, Result};
 
 /// Crate types supported by `gccrs`
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, PartialEq)]
 pub enum CrateType {
```

Comment:
You also implement `from_str` below. You should use [`FromStr`](https://doc.rust-lang.org/std/str/trait.FromStr.html) or `From<&'a str>`

Reply:
Done in 2b9d4a3